[2023-12-22 11:00:03,998] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-22T10:00:00+00:00 [queued]>
[2023-12-22 11:00:04,038] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-22T10:00:00+00:00 [queued]>
[2023-12-22 11:00:04,039] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-12-22 11:00:04,039] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2023-12-22 11:00:04,040] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-12-22 11:00:04,062] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): fill> on 2023-12-22T10:00:00+00:00
[2023-12-22 11:00:04,095] {standard_task_runner.py:52} INFO - Started process 99226 to run task
[2023-12-22 11:00:04,102] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_python_weather_v01', 'fill', '2023-12-22T10:00:00+00:00', '--job-id', '245', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_weather.py', '--cfg-path', '/tmp/tmpp3buzbgv', '--error-file', '/tmp/tmpdikfm2d4']
[2023-12-22 11:00:04,109] {standard_task_runner.py:77} INFO - Job 245: Subtask fill
[2023-12-22 11:00:04,201] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_python_weather_v01.fill 2023-12-22T10:00:00+00:00 [running]> on host 5f3722f1fc3b
[2023-12-22 11:00:04,429] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ggoni
AIRFLOW_CTX_DAG_ID=dag_with_python_weather_v01
AIRFLOW_CTX_TASK_ID=fill
AIRFLOW_CTX_EXECUTION_DATE=2023-12-22T10:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-22T10:00:00+00:00
[2023-12-22 11:00:08,790] {python.py:118} INFO - Done. Returned value was: None
[2023-12-22 11:00:08,917] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=dag_with_python_weather_v01, task_id=fill, execution_date=20231222T100000, start_date=20231222T110004, end_date=20231222T110008
[2023-12-22 11:00:09,043] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-22 11:00:09,118] {local_task_job.py:146} INFO - Task exited with return code 0
